drivers/counter/timer_tmr_cmsdk_apb.c

```diff
@@ -171,38 +173,38 @@ static const struct tmr_cmsdk_apb_cfg tmr_cmsdk_apb_cfg_0 = {
 			.flags = 0,
 			.channels = 0U,
 	},
-	.timer = ((volatile struct timer_cmsdk_apb *)DT_INST_0_ARM_CMSDK_TIMER_BASE_ADDRESS),
+	.timer = ((volatile struct timer_cmsdk_apb *)DT_INST_REG_ADDR(0)),
 	.timer_config_func = timer_cmsdk_apb_config_0,
 	.timer_cc_as = {.bus = CMSDK_APB, .state = SOC_ACTIVE,
-			.device = DT_INST_0_ARM_CMSDK_TIMER_BASE_ADDRESS,},
+			.device = DT_INST_REG_ADDR(0),},
 	.timer_cc_ss = {.bus = CMSDK_APB, .state = SOC_SLEEP,
-			.device = DT_INST_0_ARM_CMSDK_TIMER_BASE_ADDRESS,},
+			.device = DT_INST_REG_ADDR(0),},
 	.timer_cc_dss = {.bus = CMSDK_APB, .state = SOC_DEEPSLEEP,
-			 .device = DT_INST_0_ARM_CMSDK_TIMER_BASE_ADDRESS,},
+			 .device = DT_INST_REG_ADDR(0),},
 };
 
 static struct tmr_cmsdk_apb_dev_data tmr_cmsdk_apb_dev_data_0 = {
 	.load = UINT32_MAX,
 };
 
 DEVICE_AND_API_INIT(tmr_cmsdk_apb_0,
-		    DT_INST_0_ARM_CMSDK_TIMER_LABEL,
+		    DT_INST_LABEL(0),
 		    tmr_cmsdk_apb_init, &tmr_cmsdk_apb_dev_data_0,
 		    &tmr_cmsdk_apb_cfg_0, POST_KERNEL,
 		    CONFIG_KERNEL_INIT_PRIORITY_DEVICE,
 		    &tmr_cmsdk_apb_api);
 
 static void timer_cmsdk_apb_config_0(struct device *dev)
 {
-	IRQ_CONNECT(DT_INST_0_ARM_CMSDK_TIMER_IRQ_0, DT_INST_0_ARM_CMSDK_TIMER_IRQ_0_PRIORITY,
+	IRQ_CONNECT(DT_INST_IRQN(0), DT_INST_IRQN(0),
```

Comment:
This can't be right; the second argument should be a priority.  Would explain the shippable failure.

Same issue in other instances in this fille; some other files get it right.

Reply:
Good catch, now fixed.